cfme/web_ui/__init__.py

```diff
@@ -3349,7 +3349,7 @@ def select_by_value(self, value):
         index = options_map.index(value)
         if not self.is_open:
             self.open()
-        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]'.format(index)
+        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]/a'.format(index)
```

Comment:
Right, the `a` should be in every angular select so it should work with all of them?